components/eamxx/src/physics/gw/tests/infra/gw_unit_tests_common.hpp

```diff
@@ -71,64 +64,15 @@ struct UnitWrap {
     static constexpr Int max_pack_size = 16;
     static constexpr Int num_test_itrs = max_pack_size / Spack::n;
 
-    struct Base {
-      std::string     m_baseline_path;
-      std::string     m_test_name;
-      BASELINE_ACTION m_baseline_action;
-      ekat::FILEPtr   m_fid;
+    struct Base : public UnitBase {
 
       Base() :
-        m_baseline_path(""),
-        m_test_name(Catch::getResultCapture().getCurrentTestName()),
-        m_baseline_action(NONE),
-        m_fid()
+        UnitBase()
       {
-        //Functions::gw_init(); // many tests will need table data
-        auto& ts = ekat::TestSession::get();
-        if (ts.flags["c"]) {
-          m_baseline_action = COMPARE;
-        }
-        else if (ts.flags["g"]) {
-          m_baseline_action = GENERATE;
-        }
-        else if (ts.flags["n"]) {
-          m_baseline_action = NONE;
-        }
-        m_baseline_path = ts.params["b"];
-
-        EKAT_REQUIRE_MSG( !(m_baseline_action != NONE && m_baseline_path == ""),
-                          "GW unit test flags problem: baseline actions were requested but no baseline path was provided");
-
-        std::string baseline_name = m_baseline_path + "/" + m_test_name;
-        if (m_baseline_action == COMPARE) {
-          m_fid = ekat::FILEPtr(fopen(baseline_name.c_str(), "r"));
-        }
-        else if (m_baseline_action == GENERATE) {
-          m_fid = ekat::FILEPtr(fopen(baseline_name.c_str(), "w"));
-        }
+        // Functions::gw_init(); // just in case there is ever global gw data
```

Comment:
Should this comment be removed?

Reply:
In my opinion, no. If we ever need a global init, like we did in p3, this is where it should go.